app/src/ui/branches/no-pull-requests.tsx

```diff
@@ -76,6 +86,19 @@ export class NoPullRequests extends React.Component<INoPullRequestsProps, {}> {
           and get going on your next project?
         </div>
       )
+    } else if (
+      this.props.selectedPullRequest !== null &&
+      this.props.upstreamRepositoryName !== null
+    ) {
+      return (
+        <div className="call-to-action">
+          Would you like to{' '}
```

Comment:
@outofambit and I disucssed this in Slack but to sum up here: we agreed that although it might be less helpful, it makes more sense in this space for now to point to the upstream's PR page instead of linking to their exact PR.

I think this could say instead "View pull requests for [upstream repo] on GitHub"

Reply:
done in https://github.com/desktop/desktop/pull/8965/commits/b3434b547835f76ba7458effa2c175d3357f5b9d !